docs/CHANGELOG.md

```
@@ -38,6 +38,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Replace JiraUserMapping with direct Jira API user lookup and cache in Profile model(OSIDB-4865)
 
 
+### Changed
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)